docs/surveys.md

```diff
@@ -0,0 +1,111 @@
+# Survey Definitions
+
+Surveys consist of [iD](https://github.com/openstreetmap/id)-compatible presets
+alongside a metadata file.
+
+The simplest possible metadata file looks like this:
+
+```yaml
+schema_version: 0.0.1
+version: 2.4.1
+
+name: OSM
+description: Default OSM presets from iD
+```
+
+If placed alongside iD's default presets in
```

Comment:
I think this sentence is confusing. What should be placed alongside default presets? What does "placed alongside" mean? And we probably don't need to include a link to this repo anymore, we're already in the repo.